### PR TITLE
feat: Prize Tweet

### LINKED ISSUE #332: Carousel of prizes

### PR DESCRIPTION
**Pull request by Chandraprakash-Darji**
## Fixes Issue

This PR fixes the following issues:

closes #332

## Changes proposed

Here comes all the changes proposed through this PR

Add the twitter widget package 
Used carsoul package to create tweet layout

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Add all the screenshots which support your changes
image

### CODE REVIEW

**Review by siddhigate — APPROVED** [review]
Good to go 🚀

File: package.json [hunk 1]
```diff
@@ -47,7 +45,7 @@
     "husky": "^8.0.2",
     "lint-staged": "^13.1.0",
     "prettier": "^2.8.1",
-    "prettier-plugin-tailwindcss": "^0.2.1"
```
Comment (siddhigate) [inline package.json:50]: Why did you remove this package btw?

Comment (Chandraprakash-Darji) [reply package.json:50]: It is was added by mistake. It is for sorting the tailwindcss classes. As we are not using it then  not needed

Comment (siddhigate) [reply package.json:50]: okay
